backend/src/payment/providers/checkout-v3.ts

```diff
@@ -157,7 +158,7 @@ function sign(headers: {[key: string]: any}, body?: CreateRequestBody): string {
       .map((key) => [ key, headers[key] ].join(':'))
 
   const payload =  payloadArr.concat(body ? JSON.stringify(body) : '').join("\n");
-  const alg = headers['checkout-algorithm'];
+  const alg = headers['checkout-algorithm'] ? headers['checkout-algorithm'] : SIGNATURE_ALGORITHM;
```

Comment:
Could also be written as `headers['checkout-algorithm'] || SIGNATURE_ALGORITHM`, but this is fine too.

Reply:
Thanks for pointing out!